components/admin/lessons/AdminLessonCard.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import Link from 'next/link'
+import styles from '../../../scss/adminLessonCard.module.scss'
+
+type Props = {
+  heading: string
+  description: string
+  pendingFlaggedQuestions: number
+  editLessonUrl: string
+  width?: string
+  height?: string
+}
+
+export const AdminLessonCard: React.FC<Props> = ({
+  heading,
+  description,
+  pendingFlaggedQuestions,
+  editLessonUrl,
+  width,
+  height
+}) => {
+  return (
+    <div
+      className={styles.container}
+      style={{ width: `${width}`, height: `${height}` }}
```

Comment:
Is there a need for AdminLessonCard to take in width and height props?

Reply:
I was taking into consideration when designing the overall page i may need to customize the size of the component